packages/rpc-core/src/rpc-subscriptions/index.ts

```diff
@@ -48,3 +54,9 @@ export function createSolanaRpcSubscriptionsApi(config?: Config): IRpcSubscripti
         },
     });
 }
+
+export function addSolanaUnstableSubscriptionsApi<TRpcMethods>(
+    rpc: RpcSubscriptions<TRpcMethods>
+): RpcSubscriptions<TRpcMethods & SolanaRpcSubscriptionsUnstable> {
+    return rpc as RpcSubscriptions<TRpcMethods & SolanaRpcSubscriptionsUnstable>;
+}
```

Comment:
I wonder if you shouldn't actually do this in `library/src/rpc.ts`. Like, offer two different methods `createSolanaRpcSubscriptions()` and `createSolanaRpcSubscriptions_UNSTABLE()`.

That way we could reserve the right, later on, to modify the `api` we instantiate in each, without people needing to update their code in any way. Meaning: we could inject _actual_ program logic into that API to transform the subscription requests / notification responses.

Reply:
Investigating.......

Reply:
I added what you suggested!